doc/user/content/sql/system-catalog/mz_catalog.md

```diff
@@ -426,6 +427,8 @@ Field            | Type                 | Meaning
 `connection_id`  | [`text`]             | The ID of the connection associated with the source, if any. Corresponds to [`mz_connections.id`](/sql/system-catalog/mz_catalog/#mz_connections).
 `size`           | [`text`]             | The [size](/sql/create-source/#sizing-a-source) of the source.
 `envelope_type`  | [`text`]             | The [envelope](/sql/create-source/#envelopes) of the source: `none`, `upsert`, or `debezium`.
+`key_format`     | [`text`]             | The [format](/sql/create-source/#formats) of the source's key, if any.
+`value_format`   | [`text`]             | The [format](/sql/create-source/#formats) of the source, if any.
```

Comment:
This should have the same enumeration of the possible values as the sink format documentation does!

Reply:
sounds good!

Reply:
til we still support `REGEX` but don't document it